### DISCUSSION TITLE
✅ Live Results dropdown width on phone view

### DISCUSSION THREAD

**Discussion by awebguy99** [elementor/elementor#28280] [Feature Request]
### Prerequisites

- [X] I have searched for similar feature requests in both open and closed discussions and cannot find an existing request.
- [X] I have verified that the feature is still missing from the latest stable versions of Elementor or Elementor Pro.

### What problem is your feature request going to solve? Please describe..

For the new Live Results feature, when you type into the search field, a dropdown appears. Currently the width of the dropdown is the same as the width of the search field. This is fine on desktops but really restricts what can be displayed on phones. See attached;
![Drop-down-width-issue-on-phones](https://github.com/user-attachments/assets/ffd1a79c-7bcd-4921-9934-9d68017c9667)

### Describe the solution you'd like.

Please have an option to set the dropdown to full width of the device, as a responsive option;
1. So it can be the search field width for desktops.
2. And full width for phones.

### Describe alternatives you've considered.

Use a 3rd party search widget that doesn't have this issue.

### Agreement

- [X] I agree that my request may be closed without action if it doesn't meet all the requirements.

**Comment by jazir555** [comment]
Incredibly important, have had a few comments from users about this.

**Comment by awebguy99** [comment]
Any update on this please? It's quite an issue on phones.

**Comment by jazir555** [comment]
@nicholaszein?

**Comment by luxmediq** [comment]
This should be possible now with the following settings:

![image](https://github.com/user-attachments/assets/30559361-102c-4f5c-98e1-087fb72e3104)

**Comment by nicholaszein** [comment]
Hello there!

We have great news! 🙌 

### 📢 We're happy to announce that this feature was released with Elementor Pro v3.24.0! 🥳 

✅ Feel free to check it out and update your plugin to the new version!

> _Check the changelog for more information:_
> [Changelog of Elementor and Elementor Pro](https://elementor.com/help/changelog-of-elementor/)

Cheers 🥂
